src/gui/qgsmaplayersavestyledialog.h

```diff
@@ -104,6 +104,8 @@ class GUI_EXPORT QgsMapLayerSaveStyleDialog : public QDialog, private Ui::QgsMap
 
   public slots:
     void accept() override;
+    void showAll();
+    void hideAll();
```

Comment:
```suggestion
    void selectAll();
    void deselectAll();
```

These should also be private slots, not public